cmd/convox/install_test.go

```diff
@@ -308,8 +309,14 @@ func TestUrls(t *testing.T) {
 		"http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-custom-resources.html",
 	}
 
+	tr := &http.Transport{
+		TLSHandshakeTimeout: 30 * time.Second,
```

Comment:
Thirty seconds seems way to high. Maybe 5? Anything higher than that, something might be up and we should investigate.